src/app/builder/episode/episode-picker.component.ts

```diff
@@ -39,10 +39,22 @@ export class EpisodePickerComponent implements OnChanges, OnInit {
     this.select.emit(episode);
   }
 
+  private proxiedFeedUri() {
```

Comment:
since this is essentially a value based on state, I think a getter is appropriate here - it should be sufficient to change the signature to `private get proxiedFeedUri()` and change future invocations to omit the parens